modules/core/shared/src/main/scala/gem/math/Angle.scala

```diff
@@ -117,68 +115,77 @@ sealed class Angle protected (val toMicroarcseconds: Long) {
 
 object Angle extends AngleOptics {
 
-  val Angle0:   Angle = fromDegrees(0)
-  val Angle90:  Angle = fromDegrees(90)
-  val Angle180: Angle = fromDegrees(180)
-  val Angle270: Angle = fromDegrees(270)
+  /** @group Constants */ lazy val Angle0:   Angle = degrees.reverseGet(0)
+  /** @group Constants */ lazy val Angle90:  Angle = degrees.reverseGet(90)
+  /** @group Constants */ lazy val Angle180: Angle = degrees.reverseGet(180)
+  /** @group Constants */ lazy val Angle270: Angle = degrees.reverseGet(270)
```

Comment:
What does `lazy` buy us (beyond not paying the cost for creating these constants upfront)?  Every subsequent access requires a lock as I understand it.  `degrees` should be initialized by the time this runs?

Reply:
I was getting some initialization order errors, and I worry that as we accumulate static initializers elsewhere it could get worse, so I decided to be defensive. We can review lazy vals at some point but for now I'd like to leave it.